awx/ui/src/screens/Job/JobOutput/JobOutput.test.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable max-len */
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { JobsAPI, JobEventsAPI } from 'api';
```

Comment:
We could remove the JobEventsAPI import from this test since we're no longer using it. What are your thoughts on removing the JobEventsAPI from our models completely? It doesn't look like we use that model now that we have this new /jobs/:id/job_events/children_summary/ endpoint.